packages/reputation-miner/ReputationMiner.js

```diff
@@ -1079,7 +1079,7 @@ class ReputationMiner {
 
     this.miningCycleDuration = await repCycle.getMiningWindowDuration();
     this.miningCycleDuration = this.miningCycleDuration.toNumber();
-    this.constant = ethers.BigNumber.from(2).pow(256).sub(1).div(this.miningCycleDuration);
+    this.constant = ethers.constants.MaxUint256.sub(1).div(this.miningCycleDuration);
```

Comment:
I believe `MaxUint256` includes the `sub(1)`

Reply:
That's what I get for trying to rush this through :smile: